third_party/move/testing-infra/transactional-test-runner/src/framework.rs

```diff
@@ -351,17 +364,20 @@ pub trait MoveTestAdapter<'a>: Sized {
         let state = self.compiled_state();
         let (script, opt_model, warning_opt) = match syntax {
             SyntaxChoice::Source => {
+                let warnings_are_errors = run_config.get_warnings_are_errors();
```

Comment:
Why differ in how we obtain this info from `compile_module_default`? Either style to obtain `warnings_as_errors` are fine, but I would suggest using the same one consistently to help future refactorings.

Reply:
Ok, fine.